alphadia/transferlearning/train.py

```diff
@@ -564,13 +565,14 @@ def finetune_ms2(
         test_intensity_df = test_intensity_df[0]
 
         # Prepare order for peptdeep prediction
-
+        val_psm_df = refine_precursor_df(val_psm_df)
```

Comment:
could you add brief version of the PR comment here (explaining why this is required)?